prac_06/guitars.py

```diff
@@ -0,0 +1,31 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
```

Comment:
No error checking for year and cost

Reply:
you're right!
..added error checking function